### DISCUSSION TITLE
Plan and Data Collection

### DISCUSSION THREAD

**Discussion by Thatsmusic99** [plan-player-analytics/Plan#2797] [Q&A] [answered]
Hello! I've been asked to help with a project involving the usage of Plan but those in charge had some concerns with using the plugin so I wanted to cover those here, it should be relatively straightforward at least.

Is the data collected by Plan only available to those who use it (these being server owners if set up with a login prompt), or is it also sent to an external source (e.g. some plugins will use bStats for statistics), and if so, where is it sent and how is that data used?

Thank you for your time!

**Comment by AuroraLS3** [comment, marked as answer]
Hello, all data gathered about the players and servers stay in the plugin database (sqlite or mysql) unless specifically copied elsewhere by the server administrators.

Players can view their own data by default (plan.player.self permission) if they register a user. Registering a user requires a step in-game (plan.register.self permission) to do the permission check. These permission can be negated if necessary.

Plan sends the basic metrics bStats has with all plug-ins along with the database type (sqlite/mysql) and plugin version, as well as how many servers are connected to a proxy server if running on bungee. These metrics are mainly for me to monitor adoption of new versions. bStats collection can be disabled from bStats config, though I advice against it since it is useful to me for figuring out what bugs still exist in the wild with the old installations.
